settings.py

```diff
@@ -65,8 +65,8 @@
 AWS_S3_REGION_NAME = env('AWS_S3_REGION_NAME', '')
 AWS_DEFAULT_ACL = env('AWS_DEFAULT_ACL', '')
 AWS_LOCATION = env('AWS_LOCATION', '')
-AWS_STATIC = env('AWS_STATIC', False)
-AWS_MEDIA = env('AWS_MEDIA', False)
+AWS_STATIC = bool(env('AWS_STATIC', False))
+AWS_MEDIA = bool(env('AWS_MEDIA', False))
```

Comment:
My suggestion here was to change the default for these two to `bool(AWS_STORAGE_BUCKET_NAME)`, so they *default* to enabled (as they are in the current master) if `AWS_STORAGE_BUCKET_NAME` is set and are disabled otherwise.

Reply:
Oops, think I understand now. Commit coming soon.